x/payments/internal/types/msgs.go

```diff
@@ -314,8 +314,12 @@ func (msg MsgRevokeDiscount) GetSignBytes() []byte {
 }
 
 type MsgEffectPayment struct {
-	SenderDid         did.Did `json:"sender_did" yaml:"sender_did"`
-	PaymentContractId string  `json:"payment_contract_id" yaml:"payment_contract_id"`
+	SenderDid                 did.Did `json:"sender_did" yaml:"sender_did"`
+	PaymentContractId         string  `json:"payment_contract_id" yaml:"payment_contract_id"`
+	InputFromPayRemainderPool string  `json:"input_from_pay_remainder_pool" yaml:"input_from_pay_remainder_pool`
+	InputFromPayer            string  `json:"input_from_payer" yaml:"input_from_payer"`
+	OutputToPayRemainderPool  string  `json:"output_to_pay_remainder_pool" yaml:"output_to_pay_remainder_pool"`
+	OutputToPayees            string  `json:"output_to_payees" yaml:"output_to_payees"`
```

Comment:
We don't need to change the message. Please revert these changes.

Reply:
removed